cluster/addons/fluentd-gcp/fluentd-gcp-image/google-fluentd.conf

```diff
@@ -82,6 +82,14 @@
 <source>
   type tail
   format none
+  path /varlog/etcd.log
```

Comment:
No, this is adding a new section for collecting fluentd. Github just chose to display it poorly